tests/acceptance/features/bootstrap/OccContext.php

```diff
@@ -1119,6 +1134,34 @@ public function theAdminHasAddedRemovedTheApplicableUserForMountUsingTheOccComma
 		$this->theCommandShouldHaveBeenSuccessful();
 	}
 
+	/**
+	 * @When the administrator lists all the created local storage using occ command
```

Comment:
```suggestion
	 * @When the administrator lists the local storage using the occ command
```
This step just does the default list. So we do not need to mention "all".
(There is a command option `--all` and we will want another step one day to test that)